ACPs/77-reinventing-subnets/README.md

```diff
@@ -46,77 +46,154 @@ For standalone networks like the Avalanche Primary Network, this is done by conn
 
 By separating Subnet Validators from Primary Network Validators, a list of validator IPs to connect to (the functional bootstrappers of the Subnet) is no longer provided by simply connecting to the Primary Network Validators. However, the Primary Network can enable nodes tracking a Subnet to seamlessly connect to the Subnet Validators by tracking and gossiping Subnet Validator IPs. Subnets will not need to operate and maintain a set of bootstrappers and can continue to rely on the Primary Network for peer discovery.
 
-### New Registration Flow
+### Setting a Subnet Manager
 
-#### Step 1: Retrieve a BLS multisig from the Subnet
+To create a Subnet, a `CreateSubnetTx` must be issued on the P-Chain. This transaction includes an `Owner` field which defines the key that must be used to authorize any validator set additions (`AddSubnetValidatorTx`) or removals (`RemoveSubnetValidatorTx`).
 
-To provide increased optionality over Subnet Validator requirements, the P-Chain will only require an Avalanche Warp Message using the `AddressedCall` format with the below payload to join the Subnet Validator set. A `typeID` for distinguishing between the different payload types on the P-Chain must be included prior to this ACP being considered "Implementable".
+To be a Permissionless Subnet, this `Owner` key must no longer have the ability to modify the Subnet's validator set. A `ConvertSubnetTx` must first be issued to explicitly convert a Subnet from Permissioned to Permissionless. This transaction will set the `(chainID, address)` pair that will manage the Subnet going forward. After `ConvertSubnetTx` is issued, the `Owner` from the `CreateSubnetTx` that created the Subnet will no longer have the ability to modify the Subnet's validator set.
```

Comment:
Would a `[CreateChainTx`](https://github.com/ava-labs/avalanchego/blob/a0f7e422bb8497186ebff434f12c4b957cec8d49/vms/platformvm/txs/create_chain_tx.go#L36) need to be issued _before_ `ConvertSubnetTx`?

Reply:
> Would a CreateChainTx need to be issued before ConvertSubnetTx?

Yes

Reply:
> Also wondering if a Subnet can be permissionless from its creation or if CreateChainTx will eventually accept a Subnet Manager that is not Owner from the get go

Currently no. It starts as Permissioned then can be migrated to Permissionless. ACP-77 will probably stop here in terms of Subnet creation flow since it's already fairly involved. The UX streamlining you are describing would be better suited in a follow up ACP